torch/csrc/jit/codegen/cuda/compute_at_map.cpp

```diff
@@ -399,20 +399,28 @@ void IterDomainGraph::build(Fusion* fusion) {
             c_tv->domain()->domain(),
             c2f_root_map);
 
-        auto c2f_map = replay_FasC.getReplay();
-
         // Map the entire replay map between the multiple
-        // consumers even for the Parallel map as they share the same
-        // loop.
-        for (auto c_id : getSortedKeys(c2f_map, Statement::lessThan)) {
-          auto f_id = c2f_map.at(c_id);
-          // Map the id's together
-          permissive_nodes_.mapEntries(f_id, c_id);
-          exact_nodes_.mapEntries(f_id, c_id);
-          if (idIsALeafDomain(f_id, first_output_tv)) {
-            loop_nodes_.mapEntries(f_id, c_id);
+        // consumers
```

Comment:
Compute at map uses `getReplay` anyways. I'm open minded of leaving this one as `getReplay`.

Reply:
Just want to make sure I understand this correctly, there should be no difference between using `getReplay` and `getIterDomainEquivalence` in this sibling mapping case as there should be no forwarding when replaying from one sibling to another sibling.

Reply:
Put the response below, but want to put it on here as well so it doesn't look like I didn't respond...

Yes, that's correct.